app/(main)/manager/page.jsx

```
@@ -0,0 +1,82 @@
+            totalPlays: 10000
+        });
+    }, []);
+
+    return (
+        <div className="p-6">
+            <div className="flex justify-between items-center mb-6">
+                <h1 className="text-2xl font-bold">Artist Dashboard</h1>
+                <button 
+                    onClick={handleExitManager}
+                    className="text-[--md-sys-color-primary] hover:underline"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)